libmscore/rendermidi.cpp

```diff
@@ -2294,7 +2294,7 @@ void Score::renderMidi(EventMap* events, bool metronome, bool expandRepeats, con
     masterScore()->setExpandRepeats(expandRepeats);
     MidiRenderer::Context ctx(synthState);
     ctx.metronome = metronome;
-    ctx.renderHarmony = preferences.getBool(PREF_SCORE_HARMONY_PLAY);
+    ctx.renderHarmony = true; //! TODO preferences.getBool(PREF_SCORE_HARMONY_PLAY);
```

Comment:
Can you explain the meaning of this TODO?

Reply:
Remove dependency on `preferences`